lib/bn.js

```diff
@@ -731,6 +731,44 @@ BN.prototype.uxor = function uxor(num) {
 };
 
 
+// Not ``this`` with ``width`` bitwidth
+BN.prototype.inotn = function inotn(width) {
+  assert(typeof width === 'number' && width >= 0);
+
+  var bytesNeeded = Math.ceil(width / 26) | 0;
+  var bitsLeft = width % 26;
+
+  // Extend the buffer with leading zeroes
+  while (this.length < bytesNeeded) {
+    this.words[this.length++] = 0;
+  }
+
+  if (bitsLeft > 0)
+    bytesNeeded--;
+
+  // Handle complete words
+  for (var i = 0; i < bytesNeeded; i++) {
+    this.words[i] = (this.words[i] & 0xfc000000) |
```

Comment:
Of course. Thank you very much, and sorry for long comments thread on this PR! ;)